portal-frontend/src/app/features/review-application/review-submit/review-submit.component.ts

```diff
@@ -35,7 +43,14 @@ export class ReviewSubmitComponent implements OnInit, OnDestroy {
     private applicationService: ApplicationService
   ) {}
 
+  @HostListener('window:resize', ['$event'])
+  onWindowResize() {
+    this.isMobile = window.innerWidth < 480;
```

Comment:
Is it also working on a zoomed window?

Reply:
Yes, it kicks in around 300% zoom level based on window size